go/vt/wrangler/reparent.go

```diff
@@ -1085,3 +1088,23 @@ func (wr *Wrangler) TabletExternallyReparented(ctx context.Context, newMasterAli
 	}
 	return nil
 }
+
+func slaveWasRunning(stopSlaveStatus *replicationdatapb.StopSlaveStatus) bool {
+	if stopSlaveStatus == nil {
```

Comment:
This should never happen, so we should let it panic if it does happen rather than hiding the problem.

Reply:
Done.